app/views/content/subjects/engineers-teach-physics/_article.html.erb

```diff
@@ -126,7 +126,7 @@
     <h2 class="heading-s">Funding for teaching physics</h2>
 
     <p>
-      Scholarships of £29,000 or bursaries of £27,000 are available for trainee physics teachers starting their teacher training course in the 2023/24 academic year.
+      Scholarships of £29,000 or bursaries of £27,000 are available for trainee physics teachers starting their teacher training course between September 2023 and July 2024.
```

Comment:
I think this sounds a bit like if you start it in Sept 2024, you definitely won't get anything?

Last year we also added this - do we need something similar?
`Financial support for courses starting in 2023 will be announced in autumn 2022.`

(see https://github.com/DFE-Digital/get-into-teaching-app/pull/3559)